src/presence_analyzer/static/js/get_avatar.js

```diff
@@ -8,8 +8,14 @@
             $.getJSON(getAvatarUrl, function(result) {
                 $userAvatarElement.attr('src', intranetBaseUrl + result);
             }).fail(function() {
-                $userAvatarElement.attr('src', '');
-                alert('User details not found.');
+                var $errorSection = $('#error_message'), $selectedUser = $('#user_id').val();
+                if($selectedUser) {
+                    $userAvatarElement.attr('src', '');
+                    console.log($errorSection);
+                    $errorSection.html('User details not found.');
```

Comment:
text or val, not html
You should display msg in some p, span or other. not strictly in div.